UnitTests/Views/LabelTests.cs

```diff
@@ -566,4 +566,13 @@ public void Without_Top_Border ()
                                                      );
     }
 
+    [Fact]
+    public void Setting_Width_Height_Before_Set_AutoSize_To_False_Do_Not_Throws ()
+    {
+        var exception = Record.Exception (() => new Label { Width = 10, AutoSize = false });
```

Comment:
This test is inherently redundant with a constructor test, as any thrown exception is a test failure.

Doing this is trying to prove a negative, which isn't really valid.

Instead, if there is a known and expected exception, a test for that exception should be written.

Otherwise, the constructor tests do this already.

That's why they removed the Assert.DoesNotThrow method in XUnit in the first place.

Reply:
You suggesting only using the constructor and remove the `Record.Exception`? If it's that how I test the assertion?

Reply:
Actually suggesting removing this test entirely.

The known condition that would have thrown an exception before the fix should just be added to the constructor test. Un-fixed code would then fail the constructor test when the constructor is called, which is the appropriate place for it. Fixed code will pass.

Reply:
In other words, the existing constructor test was just not complete enough, if it wasn't already failing. Ensuring that case is covered makes the constructor test a stronger and more correct test from the start.

Reply:
In my opinion, they shouldn't have removed that assertion method, because there _are_ situations in which it is actually useful. This one just doesn't happen to be one of them, because the actual desired proof is that the constructor succeeds. Asserting that it does not throw is redundant, as an unhandled exception is already a failure of the test without any more work on your part.

Reply:
Done. Thanks.